get-involved.md

```diff
@@ -9,8 +9,13 @@ If you're interested to get involved in the developement then there are many way
 ## IRC
 
 The best way is to join our IRC chanel #PELUX on freenode, where most of the developers hand out.
+Join the chanel in a webbrowser:
```

Comment:
Nitpick 1: channel with two "n"s.  
Nitpick 2: Web browser should be written as two words.